conf/messages.en

```diff
@@ -58,6 +58,16 @@ factCheck.pagesReviewedConfirmationHeading = Confirm you''ve fact checked all pa
 factCheck.sendConfirmation = Confirm
 factCheck.sendConfirmationStatus =
 
+factCheckPageReview.answer.error.required = You must select one of the options shown
+factCheckPageReview.comment.error.required = No comment supplied
```

Comment:
We don't need to worry about this now, but I think the comments may be optional.

Reply:
Intentionally left comment related items which dont affect the running of the app